packages/flutter/lib/src/material/input_decorator.dart

```diff
@@ -2741,6 +2776,7 @@ class InputDecorationTheme extends Diagnosticable {
     this.disabledBorder,
     this.enabledBorder,
     this.border,
+    this.alignLabelWithHint = false,
   }) : assert(isDense != null),
        assert(isCollapsed != null),
       assert(filled != null);
```

Comment:
Give this line one more indent space and add `assert(alignLabelWithHint != null)`